oep_framework/oep/effect_player.cpp

```diff
@@ -5,6 +5,26 @@
 #include <optional>
 #include <iostream>
 
+namespace bnb::oep
+{
+
+    class js_callback : public bnb::interfaces::js_callback
```

Comment:
Can we put this declaration to the separate header in the module to reuse it in the implementations of effect_player?